person.rb

```diff
@@ -1,9 +1,11 @@
-class Person
+require './nameable'
+class Person < Nameable
   def initialize(age, parent_permission: true, name: 'Unknown')
     @id = Random.rand(1..1000)
```

Comment:
- [ ] When you pass a default string in arguments then you pass it like 
name='Unknown' :heavy_check_mark: 
Please pass your name argument as I showed above :blush: only booleans can be passed in arguments as objects as you already did :+1:

- [ ] Please change the position of arguments, they should take arguments in the way shown below :arrow_down:
`def initialize(age, name='Unknown', parent_permission: true)` :+1: